### PR TITLE
Refactor amount components and related store

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

File: src/store/modules/backend.js [hunk 1]
```diff
@@ -15,6 +15,10 @@ export default {
     tip: {},
     comment: {},
     stats: null,
+    prices: {},
+  },
+  getters: {
+    minTipAmount: ({ prices: { usd } }) => 0.01 / usd || 0.06,
```
Comment (mradkov) [inline src/store/modules/backend.js:21]: What is this `0.06` for?

The requirement was to be able to tip `0.01` USD or more 🤔

Comment (davidyuk) [reply src/store/modules/backend.js:21]: This value will be used until rates are fetched. I have just calculated it by the current rates. I think it should work fine even without a default value, should I remove that? Previously the default value was `0.01` that is also no enough clear.

Comment (mradkov) [reply src/store/modules/backend.js:21]: But the prices are already being fetched

Comment (davidyuk) [reply src/store/modules/backend.js]: I have removed that and rebased in at the top of develop.

File: src/components/FiatValue.vue [hunk 2]
```diff
@@ -16,15 +16,19 @@ export default {
...
   },
-  computed: {
-    ...mapState(['settings', 'currencyRates']),
-    fiatValue() {
-      const rate = this.currencyRates.aeternity[this.settings.currency];
-      if (!this.amount) {
-        return 0;
-      }
-      return (this.amount * rate).toLocaleString('en-US', { style: 'currency', currency: this.settings.currency });
+  computed: mapState({
```
Comment (mradkov) [inline src/components/FiatValue.vue:19]: Should we continue using this syntax?

Comment (davidyuk) [reply src/components/FiatValue.vue:19]: This is a vuex feature. They wouldn't implement that if it has an obviously bad readability or so.

**Pull request merged**